SPECIFICATIONS.md

```diff
@@ -232,6 +236,26 @@ Example:
 The verifier's check will fail because when it is evaluated, it only sees
 `right("file1", "read")` from the authority block.
 
+#### Scope annotations
+
+Rules (and blocks) can specify _trusted origins_ through a special `trusting` annotation. By default,
+only the current block and the authority block are trusted. This default can be overriden:
+
+ - at the block level
+ - at the rule level (which takes precedence over block-level annotations)
+
+The scope annotation can be either:
+
+ - `any`: this allows the rule to match facts defined by any combination of blocks. This is an unsafe setting
```

Comment:
do we really want to support `any`? which use case mandates it?

Reply:
or maybe we allow it in the authorizer but not in the token

Reply:
As far as I remember you were the only one mentioning possible use cases for it.
I'd be glad to remove it

Reply:
I've removed `any` from both the spec and the haskell impl